index.js

```diff
@@ -18,25 +16,38 @@ Fs.readdirSync(Path.join(__dirname, 'helpers')).forEach(function (file) {
 
 /**
  * Theme renderer constructor
+ * @param {Object} settings
+ * @param {Object} themeSettings
  * @param {Object} assembler
  */
-module.exports = function (assembler) {
+module.exports = function (settings, themeSettings, assembler) {
     var self = this;
 
     self.handlebars = Handlebars.create();
 
     self.handlebars.templates = {};
 
-    self.options = internals.options;
+    self.settings = settings || {};
+    self.themeSettings = themeSettings || {};
+
     self.translate = null;
     self.inject = {};
     self.helpers = [];
     self.decorators = [];
 
     _.each(Helpers, function (Helper) {
-        self.helpers.push(new Helper(self.handlebars));
+        new Helper(self.handlebars).register(self);
     });
 
+    /**
+     * Renders a string with the given context
+     * @param  {String} string
+     * @param  {Object} context
+     */
+    self.renderString = function (string, context) {
```

Comment:
Anyway we can make this a prototype method? We're going to constantly allocate memory to store this function on every request vs once with a prototype method. Same for the other ones. Perhaps outside the scope of this PR, but it would improve performance.